manifests/plugin/abrt/params.pp

```diff
@@ -0,0 +1,13 @@
+# Default parameters for the Pulp smart proxy plugin
+class foreman_proxy::plugin::abrt::params {
+  $enabled                 = true
+  $group                   = undef
+  $abrt_send_log_file      = '/var/log/foreman-proxy/abrt-send.log'
+  $spooldir                = '/var/spool/foreman-proxy-abrt'
+  $aggregate_reports       = true
+  $send_period             = 600
+  $faf_server_url          = undef
+  $faf_server_ssl_noverify = true
+  $faf_server_ssl_cert     = "${::settings::ssldir}/certs/${::clientcert}.pem"
```

Comment:
I was unaware of that settings hash. I do agree with default empty (because there is no default server URL either) or the existing class parameters.